jb/itch5/ut_order_book_depth.cpp

```diff
@@ -0,0 +1,144 @@
+#include <jb/itch5/order_book_depth.hpp>
+
+#include <boost/test/unit_test.hpp>
+
+  /* Ticket https://github.com/coryan/jaybeams/issues/20
+   *
+   * Remove old test (duplicated from ut_order_book.cpp
+   * Add tests to verify class handles Book Depth correctly
+   *
+   */
+
+BOOST_AUTO_TEST_CASE(order_book_trivial) {
+  jb::itch5::order_book_depth tested;
+  //  book_depth should be 0
+  BOOST_CHECK_EQUAL(tested.get_book_depth(), 0);  
+}
+
+BOOST_AUTO_TEST_CASE(order_book_buy) {
```

Comment:
Good test cases, please consider adding a test for a book that gets to 0 levels (i.e. all orders are removed).